.travis.yml

```diff
@@ -31,6 +31,7 @@ install:
   - go get -u -v github.com/newhook/go-symbols
   - go get -u -v golang.org/x/tools/cmd/guru
   - go get -u -v github.com/alecthomas/gometalinter
+  - go get -u -v github.com/cweill/gotests
```

Comment:
you don't need to add this to travis unless you have some unit tests. Could you add some unit tests?